client/components/AllProducts.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+
+export const AllProducts = props => {
+
+        const { products } = props;
+        return (
+        <div>
+            <div>
+                <h1>All Products</h1>
+            </div>
+            <div>
+                {
+                    products.map(product => <Link to={`/products/${product.id}`} key={product.id}><h3>{product.name}</h3></Link>)
```

Comment:
should we include a snippet of the product description and the price? I'm imaging something like this:
Item Name            Price
Description

Let me know what you think!